sharepoint/sharepoint-ps/sharepoint-online/Get-SPODataAccessGovernanceInsight.md

```diff
@@ -0,0 +1,130 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+author: pullabhk
+ms.author: pullabhk
+manager: 
+ms.reviewer:
+---
+
+# Get-SPODataAccessGovernanceInsight
+
+## SYNOPSIS
+
+Lists various 'Data Access Governance' (DAG) reports in SharePoint admin center.
+
+## SYNTAX
+
+### GetAllReportsParameterSet
+```
+Get-SPODataAccessGovernanceInsight -ReportEntity <ReportEntityEnum> [-WorkLoad <WorkloadEnum>]
+ [-ReportType <ReportTypeEnum>] [<CommonParameters>]
+```
+
+### GetReportParameterSet
+```
+Get-SPODataAccessGovernanceInsight -ReportID <Guid> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Fetch details of various Data Access Governance (DAG) reports available in SharePoint admin center.
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+Get-SPODataAccessGovernanceInsight -ReportEntity EveryoneExceptExternalUsersForItems
+```
+
+The above cmdlet fetches all DAG reports about 'Everyone except external users' attached to a item i.e., to a file, folder, or list in the last 28 days. The output consists of important parameters such as Status, ReportID, number of sites in the report and other user provided values during report generation.
+
+## PARAMETERS
+
+### -ReportEntity
+
+Specifies the entity that could cause oversharing and hence tracked by these reports. For eg: Sharing links, sharing with Everyone except external users, users with permissions etc.
```

Comment:
> For eg: Sharing links, sharing with Everyone except external users, users with permissions etc. [](http://example.com/codeflow?start=85&length=96)

remove this sentence -- the "Accepted Values" already explains what are the accepted values